staging/src/k8s.io/kube-aggregator/pkg/cmd/server/start.go

```diff
@@ -49,9 +49,7 @@ type AggregatorOptions struct {
 }
 
 // NewCommandStartAggregator provides a CLI handler for 'start master' command
-func NewCommandStartAggregator(out, err io.Writer, stopCh <-chan struct{}) *cobra.Command {
-	o := NewDefaultOptions(out, err)
-
+func NewCommandStartAggregator(o *AggregatorOptions, stopCh <-chan struct{}) *cobra.Command {
```

Comment:
This does not look right (I do not think we normally pass in options to a command).

Reply:
We can set options only by this way.

Reply:
better call `o` defaults. I don't see anything wrong with this style.

Reply:
Indicate default options in comments to prevent rename `o` to `defaults` widespread ugly.

Reply:
sgtm